internal/gcs/gcs.go

```diff
@@ -18,8 +18,14 @@ var style aurora.Aurora
 
 func IsRunningInGCP() bool {
 	resp, err := http.Get("http://metadata.google.internal")
-	resp.Body.Close()
-	return err == nil
+	
+	if err != nil {
+		return false
+	}
+
+	defer resp.Body.Close()
+
+	return true
```

Comment:
I would suggest to avoid the return `true`, and return `false` pattern
```suggestion
	if resp != nil && resp.Body != nil {
		defer resp.Body.Close()
	}
	return err == nil
```

Reply:
done